portfolio/src/main/webapp/index.html

```diff
@@ -23,15 +23,20 @@ <h1> JIMIN BYUN</h1>
       </div>
     </div>
     <div id="intro">
-      <h2>Hello! I am Jimin.</h2>
-      <p> I'm 19, rising junior at Carnegie Mellon University, School of Computer Science. 
-          I am majoring in Artificial Intelligence with additional major in Computer Science. 
-          While I enjoyed every new things I learned in school, some of best courses for me was Computer Vision, Planning Teachniques for Robotics and Parallel Algorithms. 
-          I am originally from Seoul, South Korea, although I spent most of my post elementary school years in small island called Jeju. 
-          I came to United States for the first time as a college freshman, so my experience and knowledge about US is limited to Pittsburgh, specifically within 10km radius around our school. 
-      <p> In my free time, I enjoy solving jigsaw puzzles while marathoning a movie or tv series. 
-          My friends and I spend most of our time together playing board games or going to Room Escapes. 
-          Contrary to my other hobbies, I also play drum kit for fun, especially for upbeat songs. (Not that I'm good at it)
+      <div id="intro-hand">
+        <p>👋
```

Comment:
nit: you might not need the `<p>` tag. It's perfectly fine to have just `<div id="intro-hand">👋<div>` and style it appropriately. `<p>` tags tend to add additional margins anyway, which is not something that I think you wanted.